pest/src/error.rs

```diff
@@ -18,7 +18,7 @@ use RuleType;
 use position::Position;
 use span::Span;
 
-/// A `struct` defining errors.
+/// The error type for a failed parse.
```

Comment:
I would use "parse-related" in order to include custom error messages in the definition which might be strictly failed parse errors.